force_bdss/io/tests/test_workflow_writer.py

```diff
@@ -25,9 +24,28 @@ def setUp(self):
         self.mco_parameter_factory = self.mco_factory.parameter_factories[0]
         self.data_source_factory = self.registry.data_source_factories[0]
 
+    def sample_workflow(self):
+        wf = Workflow()
+
+        wf.mco_model = self.mco_factory.create_model()
+        wf.mco_model.parameters = [self.mco_parameter_factory.create_model()]
+        wf.mco_model.kpis = [KPISpecification()]
+        wf.execution_layers = [
+            ExecutionLayer(
+                data_sources=[
+                    self.data_source_factory.create_model(),
+                    self.data_source_factory.create_model(),
+                ]
+            ),
+            ExecutionLayer(
+                data_sources=[self.data_source_factory.create_model()]
+            ),
+        ]
+        return wf
+
```

Comment:
I believe we currently don't have a `ProbeWorkflow` class or equivalent - this useful code could be refactored later into a factory for creating such an object